frontend/sw360-portlet/src/main/webapp/html/licenses/detail.jsp

```diff
@@ -59,16 +69,15 @@
                         }
                 ).render();
             }
-
     );
 
     function showWhiteListOptions() {
         Y.all('table.todosFromModerationRequest').show();
         Y.all('table.db_table').hide();
     }
+    function editLicense() {
+        window.location = '<portlet:renderURL ><portlet:param name="<%=PortalConstants.LICENSE_ID%>" value="${licenseDetail.id}"/><portlet:param name="<%=PortalConstants.PAGENAME%>" value="<%=PortalConstants.PAGENAME_EDIT%>"/></portlet:renderURL>'
+    }
 </script>
 
-
-
-
-<link rel="stylesheet" href="<%=request.getContextPath()%>/css/sw360.css">
\ No newline at end of file
+<link rel="stylesheet" href="<%=request.getContextPath()%>/css/sw360.css">
```

Comment:
Do you really need to load this stylesheet here or is it already loaded in the header? I think that it is uncommon to load a globally used stylesheet in the body